Sources/KafkaFoundationCompat/Data+KafkaContiguousBytes.swift

```diff
@@ -12,7 +12,15 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if swift(<5.9)
+@preconcurrency import Foundation
+#else
 import Foundation
+#endif
 import Kafka
 
+#if swift(<5.9)
+extension Data: @unchecked Sendable {}
+#endif
```

Comment:
Is this really required. I was hoping a `@preconcurrency import struct Foundation.Data` would be enough. I don't think we can just conform `Data` to be `Sendable` here.

Reply:
Yes this is the error I got when only doing the `@preconcurrency import`:

https://ci.swiftserver.group/job/swift-kafka-client-swift58-prb/208/console

Reply:
That errors stems from you trying to conform `Data` to `Sendable`. My assumption is we don't have to do that at all just import with pre concurrency should be enough